src/haz3lcore/dynamics/DH.re

```diff
@@ -33,6 +33,7 @@ module rec DHExp: {
     | Prj(t, int)
     | Constructor(string)
     | ConsistentCase(case)
+    | InexhaustiveCase(MetaVar.t, HoleInstanceId.t, case)
```

Comment:
Instead of having three different case constructs in DHExp, let's merge them into one with an additional flag parameter for when there is an error.